.github/workflows/quick-ci.yml

```diff
@@ -50,3 +50,18 @@ jobs:
           name: quick-ci-jvm-err
           path: '**/*_pid*.log'
           if-no-files-found: ignore
+
+  verify-python-min-version:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+          cache: 'pip'
```

Comment:
I wonder if we use 3.8 here, would it catch more 3.8 related compatibility issue?

Reply:
The project has this recommendation:

> It is recommended to use the most recent Python version to run Vermin on projects since Python's own language parser is used to detect language features, like f-strings since Python 3.6 etc.

Unless we discover otherwise, I'd like to follow this recommendation.